.github/workflows/build_and_test.yml

```diff
@@ -859,6 +859,28 @@ jobs:
                                                neondatabase/vm-compute-node-${version}:${{ needs.tag.outputs.build-tag }}
           done
 
+      - name: Azure login
+        if: github.ref_name == 'main'
+        uses: azure/login@6c251865b4e6290e7b78be643ea2d005bc51f69a  # @v2.1.1
+        with:
+          client-id: 4bb00e03-987d-4b86-a013-079729a85cb8
+          tenant-id: c8350122-1697-4543-929a-d4a75d1bb552
+          subscription-id: a57dcb0f-4249-4982-a872-1b73a7126c12
```

Comment:
Just in case, is it ok to share these values in a public repo?

Reply:
They are not secrets but I think we still shouldn't leak them.